test/unit/spec/PollController_spec.js

```diff
@@ -72,47 +72,49 @@ function (Okta, Q, OktaAuth, Util, PollingForm, Expect, Router, $sandbox, resPol
             Expect.isJsonPost($.ajax.calls.argsFor(0), {
               url: 'https://example.okta.com/api/v1/authn/cancel',
               data: {
-                stateToken: '00_J1qxqyLs-6ZutUUWfbqm-1nqnW6n2o5z2wnBRHs'
+                stateToken: '00caLtr2zn6rEXWaHyNNmTfczuToPNY2R8y3f0yAK_'
               }
             });
           });
       });
     });
   });
 
-  Expect.describe('Polling', function () {
-    describe('API', function () {
-      itp('starts polling on load', function () {
-        return setup({}, [resPolling, resPolling, resPolling, resSuccess]).then(function (test) {
-          return Expect.wait(function () {
-            return $.ajax.calls.count() > 3;
-          }, test);
-        })
-          .then(function () {
+  Expect.describe('Polling API', function () {
+    describe('called on load', function () {
+      var originalTimeout;
+      beforeEach(function () {
+        originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
```

Comment:
What does this do. Do we need this when we are not using jasmie.clock /.

Reply:
the default timeout for each test is 10000, this is just giving it some more time to execute.

Reply:
I managed to remove these 🤞